benchmark/benchmarks.jl

```diff
@@ -1,18 +1,37 @@
 using BenchmarkTools
 using Graphs
 
-DIGRAPHS = Dict{String,DiGraph}(
+const DIGRAPHS = Dict{String,DiGraph}(
     "complete100" => complete_digraph(100), "path500" => path_digraph(500)
 )
 
-GRAPHS = Dict{String,Graph}(
+const GRAPHS = Dict{String,Graph}(
```

Comment:
IIUC benchmarking is always done on all of these, and the total time is reported?

Reply:
Yes. The benchmark results are accumulated across all `Graphs` and `DiGraphs`. I find it too granular to get benchmarks for each one of the Graphs. For me, It's better to add them up.
Now the `const` is not really needed I think, since the `Graph` is always interpolated in the benchmark, but it also doesn't harm.